app/init.php

```diff
@@ -60,6 +66,10 @@
 const APP_PAGING_LIMIT = 12;
 const APP_LIMIT_COUNT = 5000;
 const APP_LIMIT_USERS = 10000;
+const APP_LIMIT_ANTIVIRUS = 20000000; //20MB
+const APP_LIMIT_ENCRYPTION = 20000000; //20MB
+const APP_LIMIT_COMPRESSION = 20000000; //20MB
+const APP_LIMIT_PREVIEW = 10000000; //10MB file size limit for preview endpoint
```

Comment:
```suggestion
const APP_LIMIT_PREVIEW = 10 * (1000 * 1000); //10MB file size limit for preview endpoint
```

Might be nice to split it like this as you have done for `APP_STORAGE_READ_BUFFER `

Reply:
I'm doing it for APP_STORAGE_READ_BUFFER because it's using MiB (1024), but this is using metric system and with comment //10 MB it's quite obvious, same for other similar vars?

Reply:
Alright makes sense 👍